src/sentry/apidocs/spectacular_ports.py

```diff
@@ -98,6 +95,17 @@ def build_choice_description_list(choices) -> str:
 drf_spectacular.plumbing.build_choice_description_list = build_choice_description_list
 
 
+def build_array_type(
+    schema: _SchemaType | None, min_length: int | None = None, max_length: int | None = None
+):
+    """
+    This function wraps the build_array_type from `drf-spectacular` because it is not type-safe.
+    """
+    if schema is None:
+        raise ValueError("Schema cannot be None")
+    return drf_build_array_type(schema=schema, min_length=min_length, max_length=max_length)
+
+
 def resolve_type_hint(hint) -> Any:
```

Comment:
for a future PR, i wonder if we can delete this now. i had to fork it for reasons that i don't think are relevant anymore